trailblazer-rails.gemspec

```diff
@@ -9,16 +9,16 @@ Gem::Specification.new do |spec|
   spec.email         = ["apotonick@gmail.com"]
 
   spec.summary       = "Convenient Rails support for Trailblazer."
-  spec.homepage      = "http://trailblazer.to/gems/trailblazer/2.0/rails.html"
+  spec.homepage      = "https://trailblazer.to/gems/trailblazer/2.0/rails.html"
```

Comment:
This should be https://trailblazer.to/2.1/docs/trailblazer.html#trailblazer-rails